requirements.txt

```diff
@@ -1,6 +1,6 @@
 #### ESSENTIAL LIBRARIES
 
-git+https://github.com/HazyResearch/snorkel.git@redux
+git+https://github.com/HazyResearch/snorkel.git@move_preprocessor
```

Comment:
note: will follow this up with a PR to change back to `redux`— this will avoid any breaking builds in the transition period

Reply:
why don't we just land redux first?

Reply:
sure, I can do that. was thinking tests/reviews might take more time, but I'll just merge them at the same time.

Reply:
also, tests will break on `tox` if this is set to redux before the merge